docs/uitour.rst

```diff
@@ -526,7 +526,8 @@ Event types:
 * ``'Loop:ChatWindowHidden'`` - User hides the chat window.
 * ``'Loop:ChatWindowDetached'`` - User detaches the chat window.
 * ``'Loop:IncomingConversation'`` - User has an incoming conversation. Event will have data boolean value ``conversationOpen`` set to ``true`` or ``false`` depending on if the chat window is open or not.
-* ``'Loop:RoomURLCopied'`` - User clicks the email or copy buttons to share a chat URL.
+* ``'Loop:RoomURLCopied'`` - User clicks the copy button to share a chat URL.
+* ``'Loop:RoomURLEmailed'`` - User clicks the email button to share a chat URL.
```

Comment:
Should we reference [the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1118346#c7) here, or trust that it'll be resolved soon?